internal/testplanet/uplink_test.go

```diff
@@ -21,196 +21,177 @@ import (
 )
 
 func TestUploadDownload(t *testing.T) {
-	ctx := testcontext.New(t)
-	defer ctx.Cleanup()
+	testplanet.Run(t, testplanet.Config{
```

Comment:
Note, some of these were specifically made to run only against sqlite to make tests run faster. But, I Jenkins might help with this.

Reply:
I needed an easy way to execute `refresh` in the discovery service during planet start, but without affecting the TestMergePlanets test, which freezes in Travis due to the discovery refresh.